tests/test.linter.js

```diff
@@ -260,17 +260,20 @@ describe('Linter.getScanner()', function() {
     assert.deepEqual(Scanner, CSSScanner);
   });
 
-  it('should return FilenameScanner', function() {
-    var addonLinter = new Linter({_: ['foo']});
-    var Scanner = addonLinter.getScanner('__MACOSX/foo.txt');
-    assert.deepEqual(Scanner, FilenameScanner);
-  });
+  const shouldBeFilenameScanned = [
+    '__MACOSX/foo.txt',
+    'wat.dll',
+    'META-INF/manifest.mf',
+  ];
+
+  for (const filename of shouldBeFilenameScanned) {
```

Comment:
Oh wait let's not change it to a var, let's change it to `let` or a `const`, both of those will rebind the variable on each iteration. If you make it `var` then you'll get one value of the loop repeated each iteration and because those are valid every time, the tests still pass! Yay, but they aren't testing the real thing.

I've put in a pull request to fix this, sorry for the numpty mistake.

https://github.com/mozilla/addons-linter/pull/674